CHANGELOG.unreleased.md

```diff
@@ -8,7 +8,7 @@
 > Users must be able to say: “Nice enhancement, I'm eager to test it”
 
 - [VM export] Feat export to `ova` format (PR [#6006](https://github.com/vatesfr/xen-orchestra/pull/6006))
-- [Backup] Add *Restore Health Check*: ensure a backup is viable by doing an automatic test restore (requires guest tools in the VM) [#6148](https://github.com/vatesfr/xen-orchestra/pull/6148)
+- [Backup] Add _Restore Health Check_: ensure a backup is viable by doing an automatic test restore (requires guest tools in the VM) [#6148](https://github.com/vatesfr/xen-orchestra/pull/6148)
```

Comment:
Please rebase on top of `master` to remove this unrelated changes.

Reply:
I always rebase on master. This change happens on commit.

This is also the reason why I changed the delimiters in the Changelog file, because they were modified automatically on commit.

Reply:
I took care of it.

> I always rebase on master. This change happens on commit.

I know, but I did the formatting on `master` directly to avoid unrelated changes in this PR :slightly_smiling_face: